src/org/ods/usecase/LeVADocumentUseCase.groovy

```diff
@@ -1105,8 +1109,10 @@ class LeVADocumentUseCase extends DocGenUseCase {
 
         def documentType = DocumentType.DTR as String
 
-        def uri = this.createOverallDocument("Overall-Cover", documentType, metadata, null, this.getWatermarkText(documentType))
-        this.updateJiraDocumentationTrackingIssue(documentType, "A new ${documentTypeName} has been generated and is available at: ${uri}.")
+        def sectionsNotDone = this.project?.data?.sectionsNotDoneDTR ? this.project.data.sectionsNotDoneDTR : []
```

Comment:
@hrcornejo there is even a shorter form for this in Groovy: `this.project.data.documents.sectionsNotDone[documentType] ?: []`. `?:` is called the Elvis operator ;-)

Reply:
Ok.